elasticsearch/documents.conf.json

```diff
@@ -37,29 +37,8 @@
         },
         "collections": {
           "properties": {
-            "facet_key":  {
-              "type" : "keyword"
-            },
             "id": {
               "type" : "keyword"
```

Comment:
added back parents, needed for the /count route : 
            "parents": {
              "type": "text",
              "fields": {
                "keyword": {
                  "type": "keyword",
                  "ignore_above": 256
                }
              }
            }